harness/setup.py

```diff
@@ -23,7 +23,7 @@
         "packaging",
         "numpy>=1.16.2",
         "psutil",
-        "pyzmq>=18.1.0",
+        "pyzmq>=22.0.0",
```

Comment:
We don't need our users to pin this so strictly?  Can we put the requirement somewhere else, that will affect CI but not users?

Reply:
Ah, makes sense. Sean had it there and I suggested the move to `setup.py`, sorry.